### PR TITLE
Topic/group approval message

### PR DESCRIPTION
**Pull request by lmacielvieira**
### Description

In order to notify the user so he can better accompany its group status, two messages should be displayed:
1 - On group joining: "Your request was sent. Please contact to approve your request within the metaspace "
2 - On ds submission to a group which you were not approved yet: "Your dataset was sent to processing. Please contact to approve your request within the metaspace , so you can visualize the group datasets."

#902 

###### RequestedAccessDialog
- [x] Created component to show group request status
- [x] Added message to be shown when requested to join group
- [x] Added message to be shown when dataset sent for processing, but user still pending approval 

###### ViewGroupPage
- [x] It shows RequestedAccessDialog when requested to join group

###### UploadPagex
- [x] It shows RequestedAccessDialog when dataset is upload to group in which user still have pending access

##### Front end
 
- [x] Unit Test
- [ ] e2e Test
 
###### Browsers and resolutions
- [x] Chrome
- [x] Safari
- [x] sm, md, lg, xl, lg

### CODE REVIEW

File: metaspace/webapp/src/modules/GroupProfile/RequestedAccessDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                {member.user.email ? <a href={`mailto:${member.user.email}`}>{member.user.name}</a>
+                  : member.user.name}
+                {memberIdx < groupAdmins.length - 2 ? ',' : ''}
+                {memberIdx === groupAdmins.length - 2 ? ' or' : ''}
+                {memberIdx === groupAdmins.length - 1 ? ' ' : ''}
+              </span>
+            },
+            )
+          }
+          to approve your request within metaspace, so you can visualize the group datasets!
```
Comment (LachlanStuart) [inline metaspace/webapp/src/modules/GroupProfile/RequestedAccessDialog.tsx:59]: "METASPACE" should always be capitalized.

File: metaspace/webapp/src/modules/GroupProfile/RequestedAccessDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          {
+            dsSubmission ? 'Your dataset was sent to processing. ' : 'Your request was sent. '
+          }
+          Please contact
+          {
+            groupAdmins.map((member: any,
+              memberIdx: number) => {
+              return <span key={member.user.id} class='ml-1'>
+                {member.user.email ? <a href={`mailto:${member.user.email}`}>{member.user.name}</a>
+                  : member.user.name}
```
Comment (LachlanStuart) [inline metaspace/webapp/src/modules/GroupProfile/RequestedAccessDialog.tsx:51]: `member.user.email` will virtually always be null BTW. Only METASPACE admins and group admins are allowed to see other users' email addresses.

**Review by LachlanStuart — APPROVED** [review]
Looks good.

What's up with the branch name though? Under [Git Flow](https://danielkummer.github.io/git-flow-cheatsheet/), PRs should either be prefixed with `feat(ure)/` (indicating merging into the development branch - `master` in our case) or `hotfix/` (indicating merging into the stable branch).

File: metaspace/webapp/src/modules/GroupProfile/RequestedAccessDialog.tsx (hunk 1 above)
Comment (lmacielvieira) [reply metaspace/webapp/src/modules/GroupProfile/RequestedAccessDialog.tsx]: Regarding the branch name, I forgot and used other standard. Thx for pointing it.

**Pull request merged**